modules/oadp-about-backup-snapshot-locations-secrets.adoc

```diff
@@ -30,20 +30,83 @@ If you use Container Storage Interface (CSI) snapshots, you do not need to speci
 
 If you use Restic, you do not need to specify a snapshot location because Restic backs up the file system on object storage.
 
+[id="create-noobaa-bucket_{context}"]
+[discrete]
+=== Create a noobaa bucket for backups
+OpenShift Data Foundation's (ODF) `noobaa` is an object storage bucket that stores VMs and containers that are backed up and waiting for migration.
+
+To define a `BackupStorageLocation' to save these objects:
+
+. Go to ODF Object Bucket Claim.
+
+[source, terminal]
+----
+apiVersion:         objectbucket.io/v1alpha1
+kind:               ObjectBucketClaim
+name:               oadp-backup-bucket
+namespace:          openshift-adp
+labels:
+app:                noobaa
```

Comment:
Indents are incorrect.

Reply:
@HagayVider1 where did you get these from? is there a source document you could reference to fix the indents here?